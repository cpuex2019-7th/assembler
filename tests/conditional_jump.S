    addi t0, zero, 1
    addi t1, zero, 1
    beq t0, t1, eq
    j neq
eq:
    addi t2, zero, 3
    j ret
neq:
    addi t2, zero, 4
    j ret
ret:
    addi a0, t2, 0 ; nop   
