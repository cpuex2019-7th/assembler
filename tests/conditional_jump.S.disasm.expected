addi t0, zero, 1
addi t1, zero, 1
beq t0, t1, 8 
jal zero, 12
addi t2, zero, 3
jal zero, 12
addi t2, zero, 4
jal zero, 4 
addi a0, t2, 0
